src/renderer/main.js

```diff
@@ -52,7 +52,7 @@ State.load(onState)
 // the dock icon and drag+drop.
 function onState (err, _state) {
   if (err) return onError(err)
-  state = _state
+  state = window.state = _state // Make available for easier debugging
```

Comment:
Interesting. The `window.state = _state` line wasn't required before.

Is this file no longer executing in the top-level scope because of the react build step?